src/Rings.jl

```diff
@@ -194,7 +194,7 @@ Test whether the ring $R$ is trivial. A ring is trivial if it consists
 of a single element, or equivalently if its characteristic is 1. Such
 rings are also called zero rings.
 """
-is_trivial(F::NCRing) = characteristic(F) == 1
+is_trivial(F::NCRing) = one(F) == zero(F)
```

Comment:
```suggestion
is_trivial(F::NCRing) = iszero(one(F))
```
Wouldn't this be better? And according to https://nemocas.github.io/AbstractAlgebra.jl/dev/ring_interface/#Basic-manipulation-of-rings-and-elements, rings should implement `iszero`

Reply:
yes